### PR TITLE
PLAT-11372: Support new UX 1.1 with garnet/WheelSliderPanel

### PR DESCRIPTION
**Pull request by SkylerBaek**
Support new UX 1.1 with garnet/WheelSliderPanel

Enyo-DCO-1.1-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### CODE REVIEW

File: garnet/src/WheelSliderControllerSample.js [hunk 1]
```diff
@@ -45,12 +32,6 @@ var WheelSliderControllerPanel = kind({
 			this.$.sampleValue.setAttribute('aria-live', 'assertive');
 		};
 	}),
-	tapCancel: function(inSender, inEvent) {
-		this.doResult({msg: 'Cancel button tapped !!'});
-	},
-	tapOK: function(inSender, inEvent) {
-		this.doResult({msg: 'OK button tapped !!'});
-	},
 	changingEventHandler: function(inSender, inEvent) {
```
Comment (ybsung) [inline garnet/src/WheelSliderControllerSample.js]: It's better to define handlers for onChange, onChanging

```
    handlers: {
        onChange: 'changeEventHandler',
        onChanging: 'changingEventHandler'
    },
```

File: garnet/src/WheelSliderControllerSample.js [hunk 2]
```diff
@@ -2,40 +2,27 @@ require('garnet');
...
+	name: 'g.sample.WheelSliderPanel',
+	kind: WheelSliderController,
 	events: {
 		onResult: ''
 	},
-	classes: 'g-common-width-height-fit g-layout-absolute-wrapper',
+	minimumValue: 0,
+	maximumValue: 100,
+	stepValue: 5,
+	value: 50,
```
Comment (ybsung) [inline garnet/src/WheelSliderControllerSample.js]: It's better to define minimumValue, maximumValue, stepValue and value in not WheeSliderPanel but g.sample.WheelSliderController

File: garnet/src/WheelSliderControllerSample.js (hunk 1 above)
Comment (SkylerBaek) [inline garnet/src/WheelSliderControllerSample.js]: @ybsung, As we discussed, all event handlers for onChange and onChanging will be removed and sample code will be written based on valueChanged().

File: garnet/src/WheelSliderControllerSample.js (hunk 2 above)
Comment (SkylerBaek) [inline garnet/src/WheelSliderControllerSample.js]: WheelSliderController always needs initial values for all properties above to work properly. So we need to guide developers to use WheelSliderController instance with initial values. Let's keep this as is.

**Comment by SkylerBaek** [comment]
@ybsung, I updated this PR as we discussed. Please review this.

**Comment by SkylerBaek** [comment]
@ybsung, a known issue is fixed.

**Pull request merged**